src/provider/hyperkzg.rs

```diff
@@ -751,7 +736,7 @@ mod tests {
       .with_fixint_encoding()
       .serialize(&proof)
       .unwrap();
-    assert_eq!(proof_bytes.len(), 368);
+    assert_eq!(proof_bytes.len(), 352);
```

Comment:
Quick q: how did the proof length change?

Reply:
Because a `Vec`s length is unknown, `bincode` adds the length of the `Vec` as a `u64` before the actual content of the data. Since this PR turned two `Vec` into fixed-sized arrays, that's two lengths that no longer need to be serialized. Removing `2\times8=16` bytes.